test/unit/cloudinary_spec.js

```diff
@@ -879,4 +879,17 @@ describe("cloudinary", function () {
     const result = cloudinary.utils.url("sample", options);
     expect(result).to.contain("$aheight_300,$mywidth_100/c_scale,h_3_mul_ih_add_$aheight,w_3_add_$mywidth_mul_3_add_4_div_2_mul_iw_mul_$mywidth");
   });
+
+  it('should support duration in video preview', () => {
```

Comment:
I would remove this test and add a test where `normalize_expression` is tested

Reply:
Fixed, Imported all the tests from the Python SDK.